### PR TITLE
UCX: fix typos

### PR DESCRIPTION
**Pull request by ivankochin**
## What
Fix existing detected misspellings.

### CODE REVIEW

**Comment by ivankochin** [comment]
codespell tool has GPL v2.0 license and if I understand the terms of license correctly, it should be OK to use it in UCX CI

**Comment by yosefe** [comment]
@ivankochin i'd suggest to split to 2 PRs:
1. fix all typos in the code - large PR
2. add spellcheck CI (and fix more typos if added since 1 is merged) - small PR

**Comment by ivankochin** [comment]
> @ivankochin i'd suggest to split to 2 PRs:
> 
> 1. fix all typos in the code - large PR
> 2. add spellcheck CI (and fix more typos if added since 1 is merged) - small PR

I removed CI job from that PR and created a new one for that purpose: https://github.com/openucx/ucx/pull/10058

File: buildlib/pr/main.yml [hunk 1]
```diff
@@ -290,7 +290,7 @@ stages:
       parameters:
         name: new
         demands: ucx_new -equals yes
-    # Temporaly disable wire-compat tests on rain machines
+    # Temporary disable wire-compat tests on rain machines
```
Comment (yosefe) [inline buildlib/pr/main.yml:293]: Temporarily

Comment (ivankochin) [reply buildlib/pr/main.yml:293]:
```suggestion
    # Temporarily disable wire-compat tests on rain machines
```

**Review by yosefe — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Comment by ivankochin** [comment]
@yosefe @brminich, sorry, I did wrong thing during squashing so your reviews were dismissed, please review one more time.

**Review by brminich — APPROVED** [review]

File: src/ucp/core/ucp_worker.c [hunk 2]
```diff
@@ -578,7 +578,7 @@ void ucp_worker_iface_activate(ucp_worker_iface_t *wiface, unsigned uct_flags)
 {
     ucp_worker_h worker = wiface->worker;
 
-    ucs_trace("activate " UCP_WIFACE_FMT " acount=%u aifaces=%u",
```
Comment (rakhmets) [inline src/ucp/core/ucp_worker.c:581]: This is not a typo. `acount` - `activate_count`. Please revert the change, or replace by `a_count`, or `activate_count`, or `activation counter`.

File: src/ucp/core/ucp_worker.c [hunk 3]
```diff
@@ -732,7 +732,7 @@ static void ucp_worker_iface_deactivate(ucp_worker_iface_t *wiface, int force)
 {
     ucp_worker_h worker = wiface->worker;
 
-    ucs_trace("deactivate " UCP_WIFACE_FMT " force=%d acount=%u aifaces=%u",
```
Comment (rakhmets) [inline src/ucp/core/ucp_worker.c:735]: Same as above.

Comment (ivankochin) [reply src/ucp/core/ucp_worker.c:735]: Nice catch, thanks, fixed

File: src/ucp/core/ucp_worker.c (hunk 2 above)
Comment (ivankochin) [reply src/ucp/core/ucp_worker.c:581]: Nice catch, thanks, fixed

**Review by yosefe — APPROVED** [review]

**Review by rakhmets — APPROVED** [review]
